chia/util/keychain.py

```diff
@@ -282,15 +378,16 @@ def delete_key_by_fingerprint(self, fingerprint: int) -> int:
         """
         removed = 0
         for index in range(MAX_KEYS + 1):
-            pkent = self._get_pk_and_entropy(get_private_key_user(self.user, index))
-            if pkent is not None:
-                pk, ent = pkent
-                if pk.get_fingerprint() == fingerprint:
+            try:
+                key_data = self._get_key_data(index)
```

Comment:
Since neither the private key nor the entropy are used, use includes_secrets=False?